packages/back-end/src/scim/users/createUser.ts

```diff
@@ -14,6 +14,7 @@ import {
 
 export function isRoleValid(role: MemberRole) {
   const validRoles: Record<MemberRole, boolean> = {
+    noaccess: true,
```

Comment:
I'm torn here - I could either update the type to OMIT `noaccess` role, but I'd need to make a reminder to myself to remove the omission after this is all live. Or, we can just add it in. Currently, only a single organization is using this, and they are not the org interested in the `no access` role, so it's likely fine to just include it.

Reply:
If we keep it as-is, technically, someone could hit this scim endpoint and create a user with the noaccess role before we've built out all of the logic.

Reply:
I think that's fine given how unlikely it is. And if it becomes an issue for an org, we can help them out of it ad-hoc